### PR TITLE
feat: add Swahili localization

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

This replaces https://github.com/freeCodeCamp/news/pull/371 to continue localization and get tests to pass.

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@larymak, here's the new PR we can use to merge your localizations.

I noticed a couple of things that I'll leave as review comments.

File: config/i18n/locales/swahili/translations.json [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
-    "see-results": "See all results for {{ searchQuery }}",
-    "no-tutorials": "No tutorials found"
+    "label": "Tafuta",
+    "accessible-name": "Wasilisha swali lako la utafutaji",
+    "placeholder": "Tafuta kati ya mafunzo zaidi ya 8,000+",
+    "see-results": "Tazama matokeo yote ya {{ searchQuery }}",
+    "no-tutorials": "Hakuna mafunzo yaliyopatikana"
   },
   "banner": {
     "default": "Learn to code — {{ <0> }}free 3,000-hour curriculum{{ </0> }}",
```
Comment (scissorsneedfoodtoo) [inline config/i18n/locales/swahili/translations.json:15]: @larymak, this string hasn't been translated yet.

Would you like to translate the current string, and have the link go to [English Learn]()?

Or would you like to adapt the string to read "Chat with other developers in Swahili", and have the link point to the [Swahili chatroom on Discord](https://discord.com/channels/692816967895220344/986667820928602162)?

**Comment by scissorsneedfoodtoo** [comment]
@larymak, also, the strings in `"footer"` (`"tax-exempt-status"`, `"mission-statement"`, and so on) haven't been translated.

Would you like to leave them in English for now? Or would it be better to translate them here?

Finally, the strings in `"fallback"` also haven't been translated. These are less important, and only show up when someones browser can't render HTML video or audio elements. But if you'd like to translate them now, just let me know what strings you'd like to add.

File: config/i18n/locales/swahili/translations.json (hunk 1 above)
Comment (larymak) [reply config/i18n/locales/swahili/translations.json:15]: I might have missed a step there, creating a PR after updating the changes, let me do that now

**Comment by scissorsneedfoodtoo** [comment]
Closed in favor of https://github.com/freeCodeCamp/news/pull/377

**Pull request closed without merging**